helpers/readme.md

```diff
@@ -6,7 +6,7 @@ Helper functions for [TzStamp] tools.
 
 ```js
 // Deno
-import { ... } from "https://gitlab.com/tzstamp/helpers/raw/0.3.0/src/mod.ts";
+import { ... } from "https://raw.githubusercontent.com/marigold-dev/tzstamp/main/helpers/mod.ts";
```

Comment:
I believe we should do a release, `0.3.1` and then put it here, using main on references like this makes so that the script may break in the future.

Reply:
This is also the case for all other URLs that contained a version before but now are pointing to main.

Reply:
I created the `0.3.2` tag because `0.3.1` version is used by tezos-merkle package. 
I replaced all uses with the `0.3.2` tag.